classes/PodsAPI.php

```diff
@@ -310,6 +310,10 @@ public function save_post_meta( $id, $post_meta = null, $strict = false, $fields
 		}
 
 		foreach ( $post_meta as $meta_key => $meta_value ) {
+
+			// Prevent WP unslash removing already sanitized input.
```

Comment:
It's possible some of this data is already sanitized/slashed when it gets to here via `PodsAPI::save_post()` which gets called from `PodsAPI::save_object()`

Reply:
That is the whole point.
WP core unslashes all data that comes through so we need to double-slash it to make suure the "intended" slashes preserve.